test/DotNetEnv.Tests/.env5

```diff
@@ -0,0 +1,14 @@
+﻿#Test Env File for Comment variations
+
+QTEST1=1234#this is a comment
+
+#Double Quoted Variables
+QTEST="1234#this is not a comment"
```

Comment:
let's not have `QTEST` (no suffix) in the middle of all the numbered `QTEST#`s, please

Reply:
In fact, this env var is never tested